### DISCUSSION TITLE
Cant install filmbox on macbook after, license registrated

### DISCUSSION THREAD

**Discussion by LegalViking** [videovillage/community#193] [General]
I have job for client in a time crunch and im not able to install plugin that has been bought for 3 months.
 Both versions cancels installation in middle of the progress in plugin manager.
How can i fix this???

**Comment by gregcotten** [comment]
You need to email us at support@videovillage.com with diagnostic logs. In the Video Village Plugins app, click Help -> Export Diagnostic Logs and email us.

**Comment by LegalViking** [comment]
There is no help part in the plugin app

**Reply by gregcotten** [reply]
There is - it's on the menu bar up top.

**Comment by LegalViking** [comment]
Where ? I dont see it anywhere, see the screenshot

**Reply by gregcotten** [reply]
It's in the system menu bar - where all menu bars on macOS go. It's at the very top of your monitor - not attached to the window.
